lib/finch.ex

```diff
@@ -165,6 +165,10 @@ defmodule Finch do
 
   @doc """
   Builds an HTTP request to be sent with `request/3` or `stream/4`.
+
+  When making HTTP/1.x requests, it is possible to send the request body in a streaming fashion.
+  In order to do so, the `body` parameter needs to take form of a tuple `{:stream, body_stream}`,
+  where `body_stream` is a `Stream` is not yet supported for HTTP/2 requests.
```

Comment:
```suggestion
  where `body_stream` is a `Stream`. This feature is not yet supported for HTTP/2 requests.
```

Reply:
I must have screwed up my previous suggestion... I think this is ready to go after this last small fix!

Reply:
accepted